### ISSUE TITLE
Email Preview show HTML view for the Plain text option

### ISSUE THREAD

**Issue by jaydeeprami** [WordImpress/Give#2514]
## Issue Overview

- In Give 2.0, Email Preview Render HTML view for the Plain text option for all Email settings.

1) For 'HTML' and 'Plain'  type Email Preview:
![screenshot-givewp dev-2017-12-13-18-31-06-263](https://user-images.githubusercontent.com/15060983/33940127-d8f22436-e033-11e7-86db-498e42a521ef.png)

## Expected Behavior

Email Preview should render Plain text when 'Email Content Type' as a 'Plain'.

## Current Behavior

- Currently, Email Preview renders Html for the both HTML and Plain Content type.

## Steps to Reproduce (for bugs)

1. Go to global settings or Per-Form email settings
2. Choose 'Email Content Type' as a 'Plain' it renders the HTML instead of the Plain text.

## Todos
- [ ] Tests
- [ ] Documentation

## WordPress Environment
<details>
```
### WordPress Environment ###

Home URL: http://givewp.dev
Site URL: http://givewp.dev
WP Version: 4.9.1
WP Multisite: –
WP Memory Limit: 256 MB
WP Debug Mode: ✔
WP Cron: ✔
Language: en_US
Permalink Structure: /%year%/%monthnum%/%day%/%postname%/
Show on Front: posts
Table Prefix Length: 3
Table Prefix Status: Acceptable
Admin AJAX: Inaccessible
Registered Post Statuses: publish, future, draft, pending, private, trash, auto-draft, inherit, refunded, failed, revoked, cancelled, abandoned, processing, preapproval

### Server Environment ###

Hosting Provider: DBH: localhost, SRV: givewp.dev
TLS Connection: 
TLS Connection: 
Server Info: nginx/1.10.1
PHP Version: ❌ 5.3.29 - We recommend a minimum PHP version of 5.6. See: PHP Requirements in Give
PHP Post Max Size: 1,000 MB
PHP Time Limit: 0
PHP Max Input Vars: 3000
PHP Max Upload Size: 1,000 MB
cURL Version: ❌ 7.38.0, OpenSSL/1.0.2h - We recommend a minimum cURL version of 7.40.
SUHOSIN Installed: –
MySQL Version: ❌ 5.5.55 - We recommend a minimum MySQL version of 5.6. See: WordPress Requirements
Default Timezone is UTC: ✔
fsockopen/cURL: ✔
SoapClient: ✔
DOMDocument: ✔
gzip: ✔
GD Graphics Library: ✔
Multibyte String: ✔
Remote Post: ❌ wp_remote_post() failed. PayPal IPN won't work with your server. Contact your hosting provider. Error: cURL error 28: Resolving timed out after 10524 milliseconds
Remote Get: ❌ wp_remote_get() failed. The Give plugin updater won't work with your server. Contact your hosting provider. Error: cURL error 28: Resolving timed out after 5518 milliseconds

### Give Configuration ###

Give Version: 2.0.0
Give Cache: Enabled
Upgraded From: 2.0.0
Test Mode: Enabled
Currency Code: INR
Currency Position: Before
Decimal Separator: .
Thousands Separator: ,
Success Page: http://givewp.dev/donation-confirmation/
Failure Page: http://givewp.dev/donation-failed/
Donation History Page: http://givewp.dev/donation-history/
Give Forms Slug: /donations/
Enabled Payment Gateways: Test Donation, Offline Donation, PayPal Standard
Default Payment Gateway: Test Donation
PayPal IPN Verification: Enabled
PayPal IPN Notifications: N/A
Admin Email Notifications: Enabled
Donor Email Access: Enabled

### Session Configuration ###

Give Use Sessions: Enabled
Session: Disabled

### Active Give Add-ons ###


### Other Active Plugins ###

Query Monitor: by John Blackbourn – 2.17.0

### Inactive Plugins ###

Give - Currency Switcher: by WordImpress – 1.0.0
Give - Email Reports: by WordImpress – 1.0.1
Give - Fee Recovery: by WordImpress – 1.3.3
Give - Form Field Manager: by WordImpress – 1.2.3
Give - Gift Aid: by WordImpress – 1.0.0
Give - PDF Receipts: by WordImpress – 2.2
Give - Recurring Donations: by WordImpress – 1.4
Give - Setting API Fields Example: by WordImpress – 1.0
Give - Stripe Gateway: by WordImpress – 1.5
Give - Tributes: by WordImpress – 1.2
WP-Cron Control: by Thorsten Ott, Erick Hitter, Automattic – 0.7.1
WP Crontrol: by John Blackbourn & contributors – 1.6.2

### Active MU Plugins ###

Local by Flywheel Relative URL (for Live Links): by Flywheel – 1.0

### Theme ###

Name: Twenty Seventeen
Version: 1.4
Author URL: https://wordpress.org/
Child Theme: No – If you're modifying Give on a parent theme you didn't build personally, then we recommend using a child theme. See: How to Create a Child Theme

```
</details>

**Comment by akashsonic** [comment]
@ravinderk  I have fixed this issue in PR #2595

**Comment by DevinWalker** [comment]
This has been resolved and somehow wasn't closed.
